force-app/main/default/classes/ServiceScheduleService.cls

```diff
@@ -94,9 +94,14 @@ public with sharing class ServiceScheduleService {
     }
 
     private List<DateTime> getDateTimes(ServiceSchedule__c schedule) {
+        if (schedule.Frequency__c == 'OneTime') {
```

Comment:
Should 'One Time' be a picklist value? (Note the space)

Reply:
Should the picklist value have a space. I was not aware so I did not add the space. I can add that if the team thinks that way

Reply:
We need to update these value checks, maybe a constant on the service would work?